src/dataflow/components/dataflow-program.tsx

```diff
@@ -523,7 +537,7 @@ export class DataflowProgram extends BaseComponent<IProps, IState> {
         }
       } else if (n.name === "Data Storage") {
         interval = n.data.interval as number;
-        datasetName = `${n.data.datasetName as string}_${GetLocalTimeStamp(programStartTime)}`;
+        datasetName = `${n.data.datasetName as string}_${getLocalTimeStamp(programStartTime)}`;
```

Comment:
Here we overwrite the file name that we presented to the user in the dialog box in `prepareToRunProgram`.